shapepipe/modules/mask_package/__init__.py

```diff
@@ -14,13 +14,16 @@
 Description
 ===========
 
-This module creates masks for bright stars, diffraction spikes, Messier
-objects, borders, and other artifacts. If a flag file is given as input,
-for example from pre-processing, the mask that is created by this module
-is joined with the mask from this external flag file. In this case
-the config flag ``USE_EXT_FLAG`` needs to be set to ``True``. To distinguish
-the newly created output flag file from the input ones, a prefix can added as
-specificed by the config entry ``PREFIX``.
+This module creates masks for bright stars, diffraction spikes, deep sky
+objects (from the Messier and NGC catalogues), borders, and other artifacts. If
+a flag file is given as input, for example from pre-processing, the mask that
+is created by this module is joined with the mask from this external flag file.
+In this case the config flag ``USE_EXT_FLAG`` needs to be set to ``True``. To
+distinguish the newly created output flag file from the input ones, a prefix
+can added as specificed by the config entry ``PREFIX``.
+
+An NGC catalogue with positions, sizes, and types is provided with
+``shapepipe``, source: www.klima-luft.de/steinicke/ngcic/rev2000/Explan.htm
```

Comment:
Missing `l` from `.htm` at the end of the link, but even then the link does not appear to work. I think this would look nicer in the docs if you simply made `NGC catalogue` clickable.